src/components/ChooseQuantity/ChooseQuantity.tsx

```diff
@@ -0,0 +1,42 @@
+import styles from "./ChooseQuantity.module.scss";
+import minusIcon from "../../assets/images/minusIcon.svg";
+import plusIcon from "../../assets/images/plusIcon.svg";
+import { useState } from "react";
+
+const ChooseQuantity = () => {
+  const [quantity, setQuantity] = useState<number>(1);
+
+  const handleDecreaseQuantity = (prevQuantity: number) => {
+    if (prevQuantity === 1) {
+      return;
+    }
+    setQuantity(prevQuantity - 1);
+  };
+
+  const handleIncreaseQuantity = (prevQuantity: number) => {
+    setQuantity(prevQuantity + 1);
```

Comment:
All these `1`s are magic numbers. I do not understand what they mean, not will you in a week from now.
Please save them into constant variables with a meaningful name/s.

Reply:
@nitzanpap  done